.teamcity/generated/LocalProject.kts

```diff
@@ -32,6 +32,7 @@ object LocalProject : Project({
 	 buildType(PushLocalWindows1903.push_local_windows_1903)
 	 buildType(PushLocalWindows1909.push_local_windows_1909)
 	 buildType(PushLocalWindows2004.push_local_windows_2004)
+	 buildType(PushStagingLinux2004_Aarch64.push_staging_linux_2004_aarch64)
```

Comment:
@burnasheva certainly! Moving forward, I think we should keep unified names for build configurations.
Specifically, I think that'd be useful for us to change `local` suffixes to `staging`, as the term `local` might be confusing: it could be (local) registry, (local) host (e.g. store image on host / artifacts), etc. 
In context of this pull request, I didn't want to mix up such changes, as, within the `main` branch, we already working on a corresponding improvements of CI/CD (including https://github.com/JetBrains/teamcity-docker-images/pull/81), including naming and the structure of the packages.